appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/visitor/OpenApiWalker.java

```diff
@@ -153,112 +153,82 @@ public void accept(ApiVisitor visitor) {
         processAnnotations(FormParam.class, visitor::visitFormParam);
 
         // All other OpenAPI annotations
-        processAnnotations(Schema.class, visitor::visitSchema, true);
-        processAnnotations(Server.class, visitor::visitServer, Servers.class, visitor::visitServers, true);
-        processAnnotations(Extensions.class, visitor::visitExtensions, true);
-        processAnnotations(Extension.class, visitor::visitExtension, true);
+        processAnnotations(Schema.class, visitor::visitSchema);
```

Comment:
I've only just noticed that Schema is processed twice. There may be a good reason for that, but out of interest does the TCK pass with either of them removed?

Reply:
No, this was unintentional. Must have pasted this wrongly.

Reply:
You were right, it had a reason. I was wrong, I did not add it, it was already present. Looked a bit into it. When removing the second one the problem is that some paths do `ApplicationProcessor.createSchema` while there already is a schema for the type whereby it gets overridden. Removing the first one however did pass all our tests and the TCK so I am going for that.